core/mobile/src/frontend/MobileApp.ts

```diff
@@ -61,7 +61,7 @@ export class MobileApp {
         authorizationClient.setAccessToken(accessToken, expirationDate);
       });
 
-      const rpcInterfaces = opts?.iModelApp?.rpcInterfaces ?? [IModelReadRpcInterface, IModelTileRpcInterface];
+      const rpcInterfaces = opts?.iModelApp?.rpcInterfaces ?? [IModelReadRpcInterface, IModelTileRpcInterface]; // eslint-disable-line deprecation/deprecation
```

Comment:
tangential; i find it weird we have default rpcs to register on Mobile but not on any other platform afaik

cc @itwin/mobilesdkadmins

Reply:
this shouldnt hold the pr up, just making a comment to keep track of this

Reply:
agreed, if we want to make that change we should do it for 4.0.  Mind creating an issue if there isn't already one?